assets/js/cache-warmup.js

```diff
@@ -1,7 +1,12 @@
 (function ($) {
     $(document).ready(function () {
 
-        var DEBUGMODE = false; // set debug mode
+        /* set debug mode */
+
+        var DEBUGMODE = false;
+        if ($('.rex-is-debugmode').length) {
```

Comment:
`body.is-debug-debugmode` evtl

Reply:
Ich würde gerne allgemeiner checken, weil man solche Klassen üblicherweise lieber ans `<html>` hängt, nicht an `<body>`. Für den Fall, dass das nochmal wechselt, muss dann hier nichts angepasst werden.

Reply:
Solange es keinen perf impact hat, ok.

Reply:
Kein Problem, solche Selektoren laufen heute ohne Performance-Auswirkungen durch.